src/main/java/me/totalfreedom/totalfreedommod/command/Command_execute.java

```diff
@@ -0,0 +1,27 @@
+package me.totalfreedom.totalfreedommod.command;
+
+import me.totalfreedom.totalfreedommod.rank.Rank;
+import me.totalfreedom.totalfreedommod.util.FLog;
+import org.apache.commons.lang3.StringUtils;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = Rank.SUPER_ADMIN, source = SourceType.ONLY_CONSOLE)
+@CommandParameters(description = "Vanilla command", usage = "/<command> <entity> <x> <y> <z> <command> OR /<command> <entity> <x> <y> <z> detect <x> <y> <z> <block> <data> <command>")
+public class Command_execute extends FreedomCommand
+{
+                
+    @Override // uses the TFM announce command base.
+    protected boolean run(CommandSender sender, Player playerSender, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        if (args.length < 1)
+        {
+            return false;
+        }
+            // instead of doing a Announce all arguments put on Execute will be logged as Warning in logs and it doesn't run any execute arguments.
```

Comment:
also, this needs to be grammatically correct..

`// Instead of announcing it, the command would rather be shown in the logs instead as a warning. It also will not run the command, nor a return statement that it failed.`